.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+<!--
```

Comment:
When does this template get used, vs. the  more specific templates like .github/ISSUE_TEMPLATE/Bug_Report.md ?

Reply:
I believe it's used if someone clicks here:

<img width="729" alt="screen shot 2018-05-11 at 4 17 22 pm" src="https://user-images.githubusercontent.com/1156625/39945028-dc8c3af0-5536-11e8-9834-47ce21250ce7.png">